cmd/kops/create.go

```diff
@@ -28,8 +30,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/kops/cmd/kops/util"
 	kopsapi "k8s.io/kops/pkg/apis/kops"
+	"k8s.io/kops/pkg/apis/kops/registry"
 	"k8s.io/kops/pkg/apis/kops/v1alpha1"
+	"k8s.io/kops/pkg/apis/kops/validation"
+	"k8s.io/kops/upup/pkg/fi"
```

Comment:
at line 280, you'll find: sshPublicKeys[**fi**.SecretNameSSHPrimary] = authorized